src/test/lambda/extensionManifest.test.ts

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import * as packageJson from '../../../package.json'
+import * as packageNlsJson from '../../../package.nls.json'
+import * as lodash from 'lodash'
+
+describe('Test Copy Lambda Func URL in extension manifest (package.json)', async () => {
+    it('title is in package.nls.json', async () => {
+        assertPackageNlsContains('AWS.generic.copyUrl')
+    })
+
+    it('is disabled in Command Palette', async () => {
+        assertContains(packageJson.contributes.menus.commandPalette, {
+            command: 'aws.copyLambdaUrl',
+            when: 'false',
```

Comment:
Don't think this test is needed. It is asserting static data.

Sometimes it's useful to assert static things if the test gives a useful "view" of complicated interlocking objects--but that is not the case here.